persist-cli/src/main/java/io/ballerina/persist/cmd/Generate.java

```diff
@@ -134,17 +144,74 @@ public void execute() {
             try {
                 BalProjectUtils.validateSchemaFile(file);
                 entityModule = BalProjectUtils.getEntities(file);
+                String fileName = entityModule.getModuleName();
```

Comment:
is this module name from read from Ballerina.toml file? We only need this validation for the module name. If not shall we move this to validate module name entered in the Ballerina.toml file

Reply:
we don't need this validation for the filename.

Reply:
updated in [566e36d](https://github.com/ballerina-platform/persist-tools/pull/148/commits/566e36dd84d453026fbf2506266f34668c08e1af)